src/solver/AugmentedMatrix.java

```diff
@@ -0,0 +1,79 @@
+package solver;
+
+import java.util.Arrays;
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
+public class AugmentedMatrix {
+    private Row[] matrix;
+
+    AugmentedMatrix(double[][] matrix) {
+        setMatrix(matrix);
+    }
+
+    AugmentedMatrix() {}
+
+    public void readMatrix(Scanner scanner) throws InputMismatchException {
+        int n = scanner.nextInt();
```

Comment:
Never use only one letter in the name. Variable should represent the value it was created for.